### PR TITLE
Issue PS-122 bug: resolve bugs inactive flow,  state district block selection

### PR DESCRIPTION
**Pull request by AkshataKatwal16**

### CODE REVIEW

File: src/components/AddNewCenters.tsx [hunk 1]
```diff
@@ -184,7 +187,27 @@ const AddNewCenters: React.FC<AddLearnerModalProps> = ({
...
+                padding: '12px 24px', // Adjust padding as needed
+                width: '200px', // Set the desired width
+              }}
+              // disabled={!submitButtonEnable}
+              // onClick={() => {
+              //   setSubmittedButtonStatus(true);
+              //   console.log("Submit button was clicked");
+              // }}
+            >
+              add
```
Comment (itsvick) [inline src/components/AddNewCenters.tsx:206]: @AkshataKatwal16 translate

File: src/components/FormControl.tsx [hunk 2]
```diff
@@ -58,7 +62,7 @@ const MultipleSelectCheckmarks: React.FC<MultipleSelectCheckmarksProps> = ({
         <Select
           labelId="multiple-checkbox-label"
           id="multiple-checkbox"
-          value={selectedCategories}
+          value={selectedCategories?.length===0  || selectedCategories[0]===""?defaultValue?[defaultValue]:"":selectedCategories}
```
Comment (itsvick) [inline src/components/FormControl.tsx:65]: @AkshataKatwal16 format code

File: src/components/layouts/header/Profile.tsx [hunk 3]
```diff
@@ -135,9 +163,15 @@ const Profile = () => {
             >
               Logout
```
Comment (itsvick) [inline src/components/layouts/header/Profile.tsx:164]: @AkshataKatwal16 translate

File: src/components/AreaSelection.tsx [hunk 4]
```diff
@@ -96,7 +98,9 @@ const AreaSelection: React.FC<DropdownBoxProps> = ({
             tagName={t("FACILITATORS.STATE")}
             selectedCategories={selectedState}
             onCategoryChange={handleStateChangeWrapper}
-            overall={isCenterSelection ? false : true}
+            overall={inModal ? false : true}
```
Comment (itsvick) [inline src/components/AreaSelection.tsx:101]: @AkshataKatwal16 inModal is already a boolean right?

Comment (AkshataKatwal16) [reply src/components/AreaSelection.tsx:101]: yes

**Pull request merged**